sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-b2c-oidc/src/main/java/com/azure/spring/sample/aad/b2c/security/WebSecurityConfiguration.java

```diff
@@ -20,9 +20,8 @@ public WebSecurityConfiguration(AADB2COidcLoginConfigurer configurer) {
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         http.authorizeRequests()
-                .antMatchers("/login").permitAll()
                 .anyRequest().authenticated()
                 .and()
-            .apply(configurer);
+                .apply(configurer);
```

Comment:
Should the `.apply` have the same indent as `.authorizeRequests`? If so, we could add `// @formatter:off` to avoid the IDEA formatting it automatically.